### PR TITLE
[Meta Collection] add ansible.network network ee jobs

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>

### CODE REVIEW

**Comment by rohitthakur2590** [comment]
@pabelanger  Please review.

File: zuul.d/ansible-network-jobs.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+- job:
+    name: network-ee-integration-tests-network
```
Comment (pabelanger) [inline zuul.d/ansible-network-jobs.yaml:3]: we should rename this to network-ee-integration-tests-ansible-network

To be more specific, ideally we do that for vyos too. I'll propose a change later for that

File: zuul.d/project-templates.yaml [hunk 2]
```diff
@@ -1443,6 +1443,19 @@
     gate:
       jobs: *network-ee-test-jobs
 
+- project-template:
+    name: ansible-collections-ansible-network
+    check:
+      jobs: &ansible-collections-ansible-network-jobs
+        - network-ee-integration-tests-network
+        - build-ansible-collection:
```
Comment (pabelanger) [inline zuul.d/project-templates.yaml:1451]: we don't need build-ansible-collection here

File: zuul.d/ansible-network-jobs.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+- job:
+    name: network-ee-integration-tests-ansible-network
+    parent: ansible-network-vyos-appliance
+
+- job:
+    name: network-ee-integration-tests-ansible-network
```
Comment (pabelanger) [inline zuul.d/ansible-network-jobs.yaml:7]: ha, we should move set collection_namespace / collection_name here along with skip_tag. Then only set libssh below

Comment (pabelanger) [reply zuul.d/ansible-network-jobs.yaml:7]: actually, I'm not sure we need skip_tags

File: zuul.d/ansible-network-jobs.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      test_fips_mode: false
+    nodeset: vyos-1.1.8-python38
+
+- job:
+    name: network-ee-integration-tests-ansible-network-libssh
+    parent: network-ee-integration-tests-ansible-network
+    vars:
+      test_ansible_network_cli_ssh_type: libssh
+      # NOTE(pabelanger): We can only use RSA ssh keys for vyos.
+      test_fips_mode: false
```
Comment (pabelanger) [inline zuul.d/ansible-network-jobs.yaml:23]: this can be dropped, we pull it in from network-ee-integration-tests-ansible-network

Comment (rohitthakur2590) [reply zuul.d/ansible-network-jobs.yaml]: Updated. Thanks

**Review by GomathiselviS — APPROVED** [review]

**Pull request merged**